Dockerfile

```diff
@@ -3,7 +3,7 @@
 #   for the development image, look at the LDMX-Software/docker repo
 ###############################################################################
 
-FROM ldmx/dev:v5.1.0
+FROM ldmx/dev:v5.1.1
 
 # install ldmx-sw into the container at /usr/local
 COPY . /code
```

Comment:
This will fail to build since I removed `/etc/entry.sh` but the good news is that we don't need it anymore. I'll patch this branch manually and then merge it.